### ISSUE TITLE
Add `papis cache` command

### ISSUE THREAD

**Issue by alejandrogallo** [papis/papis#589]
Right now the cache is controlled by the sole flag `--clear-cache`
and so on.

I think it would be quite good for plugin writers if they could for instance
invalidate or refresh the cache of single documents.
Let's say you edit the info.yaml file from a plugin, and you want papis
to reload the information of the document but not clear the whole cache, so you would do

```
papis cache refresh papis_id:654984984949
```

then papis should load the info file of the document and refresh the cache.

The functionality I envision

```
papis cache update [query] [--all] [--doc-folder]          # Refresh the cache of already added docuements
papis cache add [--doc-folder]                              # Add a new document to the cache
papis cache pwd                                             # Print path to the current cache (~/.cache/papis...)
```

What else could be useful for this?
Maybe @jghauser  can share some thoughts about the connection of this with the nvim plugin.

**Comment by jghauser** [comment]
That does sound very useful, I think currently papis.nvim just relies on the user clearing the cache to get the papis database updated when editing info.yaml files (the papis.nvim database is independent and will be updated on file changes). So yes, this would definitely help!

I also really like the `papis cache add --doc-folder` command, which will be useful to force generation of `papis_id`s in a script I use to manually add entries.  :)

**Comment by alejandrogallo** [comment]
I did an implementation, I will be making a pull request shortly ;)

**Comment by alexfikl** [comment]
Fixed by #603 :heart:
